SYCL/Basic/alloc_pinned_host_memory.cpp

```diff
@@ -37,6 +37,7 @@ int main() {
   }
 }
 
+// CHECK:---> piMemBufferCreate
 // CHECK:---> piMemBufferCreate
 // CHECK-NEXT: {{.*}} : {{.*}}
-// CHECK-NEXT: {{.*}} : 9
+// CHECK-NEXT: {{.*}} : 17
```

Comment:
What does this number represent and why it changed?

Reply:
this is the 'Flags' parameter value for piMemBufferCreate. It has chenged because the test is now tracking the trace of the second piMemBufferCreate call (with the use_pinned_host_memory property).
---> piMemBufferCreate(                                                                                                                                         
     <unknown> : 0xab0540                                                                                                                                         
     <unknown> : 17                                                                                                                                                       
     <unknown> : 40                                                                                                                                                      
     <unknown> : 0                                                                                                                                                         
     <unknown> : 0x7ffda2fa5b08                                                                                                                              
     <nullptr>

Reply:
I see, thanks. LGTM.